cron_test_prs_jsc-zen2.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# change to directory in which this script is located
+cd `dirname $(realpath $0)`
+
+# update EasyBuild to latest 'develop' branches
+./easybuild_develop.sh
+
+# set up environment
+EB_PREFIX=$HOME/easybuild source init_env_easybuild_develop.sh
+
+# check notifications for new PRs to test
+./test_prs_jsc-zen2.sh
```

Comment:
Should we just tweak the existing `cron_test_prs.sh` to accept a required argument for which `test_prs_*.sh` script to run?

```shell
#!/bin/bash

if [ $# -ne 1 ]; then
    echo "Usage: $0 <test_prs_*.sh script to run>" >&2
    exit 1
fi
TEST_PRS_SCRIPT=$1

...

# check notifications for new PRs to test
$TEST_PRS_SCRIPT

Reply:
Thanks for the suggestion! I like the idea.

Done in https://github.com/boegel/boegelbot/pull/10/commits/89a7c2877016e3ea3eaa5c3f655fe46e3930a065